
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">


<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
  <head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    
    <meta name="generator" content="Bluefish 2.2.2" />
    <meta name="iWeb-Build" content="local-build-20100727" />
    <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7" />
    <meta name="viewport" content="width=700" />
    <title>GenoSNP</title>
    <link rel="stylesheet" type="text/css" media="screen,print" href="genosnp/GenoSNP.css" />
    <script type="text/javascript" src="genosnp/iWebSite.js"></script>
    <script type="text/javascript" src="genosnp/Widgets/SharedResources/WidgetCommon.js"></script>
    <script type="text/javascript" src="genosnp/Widgets/Navbar/navbar.js"></script>
    <script type="text/javascript" src="genosnp/iWebImage.js"></script>
    <script type="text/javascript" src="genosnp/GenoSNP.js"></script>
  </head>
  <body style="background: rgb(101, 101, 101); margin: 0pt; " onload="onPageLoad();" onunload="onPageUnload();">
    <div style="text-align: center; ">
      <div style="margin-bottom: 10px; margin-left: auto; margin-right: auto; margin-top: 10px; overflow: hidden; position: relative; word-wrap: break-word;  background: rgb(255, 255, 255); text-align: left; width: 700px; " id="body_content">
        <div style="margin-left: 0px; position: relative; width: 700px; z-index: 0; " id="nav_layer">
          <div style="height: 0px; line-height: 0px; " class="bumper"> </div>
          <div class="com-apple-iweb-widget-navbar flowDefining" id="widget0" style="margin-left: 20px; margin-top: 0px; opacity: 1.00; position: relative; width: 660px; z-index: 1; ">
    
            <div id="widget0-navbar" class="navbar">

      
              <div id="widget0-bg" class="navbar-bg">

        
                <ul id="widget0-navbar-list" class="navbar-list">
 <li></li> 
</ul>
                
      
</div>
              
    
</div>
          </div>
          <script type="text/javascript"><!--//--><![CDATA[//><!--
new NavBar('widget0', 'Scripts/Widgets/Navbar', 'Scripts/Widgets/SharedResources', '.', {"path-to-root": "", "navbar-css": ".navbar {\n\tfont-family: 'Helvetica Neue', Arial, sans-serif;\n\tfont-size: .8em;\n\tcolor: #666666;\n\tline-height: 30px;\n\tborder-bottom: 3px solid #ccc;\n}\n\n.navbar-bg {\n\ttext-align: right;}\n\n.navbar-bg ul {\n\tlist-style: none;\n\tmargin: 0px;\n\tpadding: 0px;\n}\n\n\nli {\n\tlist-style-type: none;\n\tdisplay: inline;\n\tpadding: 0px 5px 0px 0px;\n}\n\n\nli a {\n\ttext-decoration: none;\n\tpadding: 10px;\n\tcolor: #666666;\n\tfont-weight: bold;\n}\n\nli a:visited {\n\ttext-decoration: none;\n\tpadding: 10px;\n\tcolor: #666666;\n\tfont-weight: bold;\n}\n\nli a:hover\r{\r\n \tcolor: #999999;\n\ttext-decoration: none;\r}\n\n\nli.current-page a\r{\r\t color: #66ABC5;\n\ttext-decoration: none;\r}", "current-page-GUID": "202937A9-B764-4A86-8EBB-D8E06FDD6A13", "isCollectionPage": "NO"});
//--><!]]></script>
          <div style="clear: both; height: 0px; line-height: 0px; " class="spacer"> </div>
        </div>
        <div style="height: 75px; margin-left: 0px; position: relative; width: 700px; z-index: 10; " id="header_layer">
          <div style="height: 0px; line-height: 0px; " class="bumper"> </div>
          <div id="id1" style="height: 43px; left: 20px; position: absolute; top: 17px; width: 158px; z-index: 1; " class="style_SkipStroke shape-with-text">
            <div class="text-content style_External_158_43" style="padding: 0px; ">
              <div class="style">
                <p style="padding-bottom: 0pt; padding-top: 0pt; " class="Header">GenoSNP</p>
              </div>
            </div>
          </div>
        </div>
        <div style="margin-left: 0px; position: relative; width: 700px; z-index: 5; " id="body_layer">
          <div style="height: 0px; line-height: 0px; " class="bumper"> </div>
          <div style="height: 1px; line-height: 1px; " class="tinyText"> </div>
          <div style="margin-left: 28px; margin-top: 1805px; position: relative; width: 605px; z-index: 1; " class="style_SkipStroke_1 shape-with-text flowDefining">
            <div class="text-content style_External_605_66" style="padding: 0px; ">
              <div class="style_1">
                <p style="padding-top: 0pt; " class="Body">[1].    A Variational Bayes within-sample SNP genotyping algorithm that does not require a reference population.<br /></p>
                <p class="Body">        Giannoulatou E, Yau C, Colella S, Ragoussis J, Holmes CC. <a title="http://bioinformatics.oxfordjournals.org/" href="http://bioinformatics.oxfordjournals.org/">Bioinformatics.</a> <a title="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/24/19/2209" href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/24/19/2209">2008 Oct 1;24(19):2209-14</a><br /></p>
                <p class="Body"><br /></p>
              </div>
              <div style="height: 15px; line-height: 15px; " class="tinyText"> </div>
            </div>
          </div>
          


          <div style="height: 210px; width: 210px;  height: 210px; left: 456px; position: absolute; top: 13px; width: 210px; z-index: 1; " class="tinyText style_SkipStroke_2 stroke_0">
            <img src="genosnp/Oxford_University_Logo.jpg" alt="" style="border: none; height: 210px; width: 210px; " />
          </div>
          


          <div id="id2" style="height: 26px; left: 34px; position: absolute; top: 308px; width: 88px; z-index: 1; " class="style_SkipStroke_3 shape-with-text">
            <div class="text-content Normal_External_88_26" style="padding: 0px; ">
              <div class="Normal">
                <p style="padding-bottom: 0pt; padding-top: 0pt; " class="paragraph_style">Download</p>
              </div>
            </div>
          </div>
          


          <div id="id3" style="height: 22px; left: 28px; position: absolute; top: 1766px; width: 99px; z-index: 1; " class="style_SkipStroke_3 shape-with-text">
            <div class="text-content Normal_External_99_22" style="padding: 0px; ">
              <div class="Normal">
                <p style="padding-bottom: 0pt; padding-top: 0pt; " class="paragraph_style">References</p>
              </div>
            </div>
          </div>
          


          <div id="id4" style="height: 291px; left: 31px; position: absolute; top: 9px; width: 396px; z-index: 1; " class="style_SkipStroke_1 shape-with-text">
            <div class="text-content style_External_396_291" style="padding: 0px; ">
              <div class="style_1">
                <p style="padding-top: 0pt; " class="paragraph_style_1">GenoSNP is a genotyping algorithm for the Illumina Infinium SNP genotyping assay [1]. It is entirely within-sample and does not require the need for a population of control samples nor parameters derived from such a population. The ability to call genotypes using only within-sample information makes the method computationally light and practical for studies involving rare variants and small sample sizes.<br /></p>
                <p class="paragraph_style_1"><br /></p>
                <p style="padding-bottom: 0pt; " class="paragraph_style_1">GenoSNP works on Illumina Infinium raw intensity data. It can be used for genotyping SNPs from HumanHap300Duo and Duo+, Hap240S, HumanHap550, HumanHap550-Duo, HumanHap650Y and Human1M BeadChip data. It works by clustering on the intensities for each BeadPool (or BeadSet) separately. Therefore the BeadPool information for all the SNPs is necessary. The BeadPool information is the .bpm file for the array (provided by Illumina) but you can also find it in the SNP table of GenomeStudio under “Norm ID”. There are 24 different BeadPools for the Human-1, 14 for the Hap240S, 11 for the Hap300, 25 for the Hap550, and 27 for the Hap650Y.</p>
              </div>
            </div>
          </div>
          


          <div id="id5" style="height: 21px; left: 31px; position: absolute; top: 471px; width: 240px; z-index: 1; " class="style_SkipStroke_3 shape-with-text">
            <div class="text-content Normal_External_240_21" style="padding: 0px; ">
              <div class="Normal">
                <p style="padding-bottom: 0pt; padding-top: 0pt; " class="paragraph_style">Command Line Arguments</p>
              </div>
            </div>
          </div>
          


          <div id="id6" style="height: 29px; left: 31px; position: absolute; top: 700px; width: 240px; z-index: 1; " class="style_SkipStroke_3 shape-with-text">
            <div class="text-content Normal_External_240_29" style="padding: 0px; ">
              <div class="Normal">
                <p style="padding-bottom: 0pt; padding-top: 0pt; " class="paragraph_style">Input Data Formats</p>
              </div>
            </div>
          </div>
          


          <div id="id7" style="height: 27px; left: 28px; position: absolute; top: 1191px; width: 220px; z-index: 1; " class="style_SkipStroke_3 shape-with-text">
            <div class="text-content Normal_External_220_27" style="padding: 0px; ">
              <div class="Normal">
                <p style="padding-bottom: 0pt; padding-top: 0pt; " class="paragraph_style">Output Data Formats</p>
              </div>
            </div>
          </div>
          


          <div id="id8" style="height: 42px; left: 26px; position: absolute; top: 1594px; width: 379px; z-index: 1; " class="style_SkipStroke_3 shape-with-text">
            <div class="text-content Normal_External_379_42" style="padding: 0px; ">
              <div class="Normal">
                <p style="padding-bottom: 0pt; padding-top: 0pt; " class="paragraph_style">MatLab Code and plotting functions in R</p>
              </div>
            </div>
          </div>
          


          <div id="id9" style="height: 204px; left: 28px; position: absolute; top: 503px; width: 553px; z-index: 1; " class="style_SkipStroke_4 shape-with-text">
            <div class="text-content graphic_textbox_layout_style_default_External_553_204" style="padding: 0px; ">
              <div class="graphic_textbox_layout_style_default">
                <p style="padding-top: 0pt; " class="Free_Form">./GenoSNP -snps snpfile.txt -samples samplefile.txt -cutoff 0.7 -calls calls.txt -probs probabilities.txt<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form">Input parameters. Both SNPs and Samples files are mandatory:<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form">-snps        mandatory SNPs input file<br /></p>
                <p class="Free_Form">-samples  mandatory Samples input file<br /></p>
                <p class="Free_Form">-cutoff      optional cutoff, default is 0.7 <br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form">Output parameters. The user may optionally choose for the raw probabilities to be output.<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form">-calls        mandatory Calls output file<br /></p>
                <p class="Free_Form">-probs      optional Probabilities output file<br /></p>
              </div>
            </div>
          </div>
          


          <div id="id10" style="height: 118px; left: 29px; position: absolute; top: 342px; width: 570px; z-index: 1; " class="style_SkipStroke_4 shape-with-text">
            <div class="text-content graphic_textbox_layout_style_default_External_570_118" style="padding: 0px; ">
              <div class="graphic_textbox_layout_style_default">
                <p style="padding-top: 0pt; " class="Free_Form">GenoSNP can be downloaded for academic research use only. For commercial or other use please contact the authors. Copyright is retained by the University of Oxford.<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form">You can download the C code from <a title="genosnp/GenoSNP_Src_v1.3.zip" href="genosnp/GenoSNP_Src_v1.3.zip">here</a>.<br /></p>
                <p class="Free_Form">A statically linked executable file for Linux (x86_64) is provided <a title="genosnp/GenoSNP_Exe_v1.3.zip" href="genosnp/GenoSNP_Exe_v1.3.zip">here</a>.<br /></p>
                <p class="Free_Form"><br /></p>
                <p style="padding-bottom: 0pt; " class="Free_Form">Please address all questions and comments to Eleni Giannoulatou at <a title="mailto:E.Giannoulatou@victorchang.edu.au" href="mailto:E.Giannoulatou@victorchang.edu.au">E.Giannoulatou@victorchang.edu.au</a></p>

              </div>
            </div>
          </div>
          


          <div id="id11" style="height: 458px; left: 25px; position: absolute; top: 732px; width: 621px; z-index: 1; " class="style_SkipStroke_4 shape-with-text">
            <div class="text-content graphic_textbox_layout_style_default_External_621_458" style="padding: 0px; ">
              <div class="graphic_textbox_layout_style_default">
                <p style="padding-top: 0pt; " class="Free_Form">The input and output formats should be tab-separated and have no header line. Values that are closely coupled, such as AlleleA/AlleleB, RawX/RawY or probAA/probAB/probBB, could be stored in the same cell, separated by single spaces.<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="paragraph_style_2">SNPs file<br /></p>
                <p class="Free_Form">The SNPs input file describes the indexes of columns in the Samples file. It also records the Bead Pool and the Alleles of each SNP.<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form">A tab-separated file with one SNP per row, and alleles separated by a single space.<br /></p>
                <p class="Free_Form">&lt;SNP_ID&gt;[TAB]&lt;BEADSET_ID&gt;[TAB]&lt;ALLELE_A&gt;[SP]&lt;ALLELE_B&gt;<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form">For example:<br /></p>
                <p class="Free_Form">rs1256528    225    T G<br /></p>
                <p class="Free_Form">rs2905062    224    T C<br /></p>
                <p class="Free_Form">rs1815606    223    A G<br /></p>
                <p class="Free_Form">rs6603811    225    A G<br /></p>
                <p class="Free_Form">rs7531583    225    A C<br /></p>
                <p class="Free_Form"> <br /></p>
                <p class="paragraph_style_2">Samples file<br /></p>
                <p class="Free_Form">The Samples input file is the core data file containing the RawX/RawY values for each Sample. The SNP columns are indexed as per SNPs file. If only Individual ID is required then the user should duplicate the Individual ID for the Family ID.<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form">A tab-separated file with one row per Sample, and the rawX, rawY values separated by a single space.<br /></p>
                <p class="Free_Form">&lt;FAMILY_ID&gt;[TAB]&lt;INDIVIDUAL_ID&gt;[TAB]&lt;RAW_X_SNP_1&gt;[SP]&lt;RAW_Y_SNP_1&gt;[TAB]&lt;RAW_X_SNP_2&gt;[SP]&lt;RAW_Y_SNP_2&gt;    ...and so on as per SNPs file...<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form">For example:<br /></p>
                <p class="Free_Form">WTD0001-O-A08    WTD0001-O-A08    15999 10834    1284 17773    20689 606      ...<br /></p>
                <p class="Free_Form">WTD0001-O-B01    WTD0001-O-B01    15600 31048    1528 12968    19610 2118    ...<br /></p>
                <p class="Free_Form">WTD0001-O-B04    WTD0001-O-B04    15600 21048    1528 12968    19610 2118    ...<br /></p>
              </div>
            </div>
          </div>
          


          <div id="id12" style="height: 359px; left: 24px; position: absolute; top: 1228px; width: 651px; z-index: 1; " class="style_SkipStroke_4 shape-with-text">
            <div class="text-content graphic_textbox_layout_style_default_External_651_359" style="padding: 0px; ">
              <div class="graphic_textbox_layout_style_default">
                <p style="padding-top: 0pt; " class="paragraph_style_2">Calls file<br /></p>
                <p class="Free_Form">The Calls output file contains the resulting genotype for each SNP. 'No calls' are marked by 0 0. Otherwise the Alleles are carried over from the SNPs file.<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form">A tab-separated file with one row per Sample, and the allele A &amp; B values separated by a single space. <br /></p>
                <p class="Free_Form">&lt;FAMILY_ID&gt;[TAB]&lt;INDIVIDUAL_ID&gt;[TAB]&lt;ALLELE_A_SNP_1&gt;[SP]&lt;ALLELE_B_SNP_1&gt;[TAB]&lt;ALLELE_A_SNP_2&gt;[SP]&lt;ALLELE_B_SNP_2&gt;    ...and so on as per SNP file...<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form">For example:<br /></p>
                <p class="Free_Form">WTD0001-O-A08    WTD0001-O-A08    0 0    A G    A A    G G    C G    ...<br /></p>
                <p class="Free_Form">WTD0001-O-B01    WTD0001-O-B01    A A    G G    C G    T C    0 0    ...<br /></p>
                <p class="Free_Form">WTD0001-O-B04    WTD0001-O-B04    A A    G G    C G    T C    0 0    ...<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="paragraph_style_2">Probabilities file<br /></p>
                <p class="Free_Form">The Probabilities output file outputs the all three probabilities for the possible genotypes for each SNP.<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form">A tab-separated file with one row per Sample, and the probabilities AA, AB, BB separated by single spaces.<br /></p>
                <p class="Free_Form">&lt;FAMILY_ID&gt;[TAB]&lt;INDIVIDUAL_ID&gt;[TAB]&lt;PROB_AA_SNP_1&gt;[SP]&lt;PROB_AB_SNP_1&gt;[SP]&lt;PROB_BB_SNP_1&gt;[TAB]&lt;PROB_AA_SNP_2&gt;[SP]&lt;PROB_AB_SNP_2&gt;[SP]&lt;PROB_BB_SNP_2&gt;    ...and so on as per SNP file...<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form">For example:<br /></p>
                <p class="Free_Form">WTD0001-O-A08    WTD0001-O-A08    0.999212 0.000716231 7.20932e-05    0.999212 0.000716231 7.20932e-05    ...<br /></p>
                <p class="Free_Form">WTD0001-O-B01    WTD0001-O-B01    0.999991 5.65942e-06 3.80093e-06    0.999991 5.65942e-06 3.80093e-06    ...<br /></p>
                <p style="padding-bottom: 0pt; " class="Free_Form">WTD0001-O-B04    WTD0001-O-B04    0.999991 5.65942e-06 3.80093e-06    0.999991 5.65942e-06 3.80093e-06    ...</p>
              </div>
            </div>
          </div>
          


          <div id="id13" style="height: 100px; left: 21px; position: absolute; top: 1655px; width: 619px; z-index: 1; " class="style_SkipStroke_4 shape-with-text">
            <div class="text-content graphic_textbox_layout_style_default_External_619_100" style="padding: 0px; ">
              <div class="graphic_textbox_layout_style_default">
                <p style="padding-top: 0pt; " class="Free_Form">The MatLab version of GenoSNP is available <a title="genosnp/GenoSNP_MatLab.zip" href="genosnp/GenoSNP_MatLab.zip">here</a>. Please consult the included README file for input/output data formats.<br /></p>
                <p class="Free_Form"><br /></p>
                <p class="Free_Form">You can also find some R functions for plotting <a title="genosnp/R_PlottingFunctions.zip" href="genosnp/R_PlottingFunctions.zip">here</a>. These use the previous version for the format of the Input/Output so please consult the included README file.<br /></p>
              </div>
            </div>
          </div>
          <div style="height: 0px; line-height: 0px; " class="spacer"> </div>
        </div>
        <div style="height: 75px; margin-left: 0px; position: relative; width: 700px; z-index: 15; " id="footer_layer">
          <div style="height: 0px; line-height: 0px; " class="bumper"> </div>
        </div>
      </div>
    </div>
  </body>
</html>


