<!DOCTYPE HTML>
<html>

    <head>
	<meta charset="utf-8" />
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
	<meta name="description" content="Yau Group">
	<meta name="author" content="Christopher Yau">
	<meta name="generator" content="Hugo 0.51" />
	<title>Research &middot; Yau Group</title>
	<!-- Stylesheets -->
	
	<link rel="stylesheet" href="https://cwcyau.github.io/css/main.css"/>
	
	

	

	<!-- Custom Fonts -->
	<link href="https://cwcyau.github.io/css/font-awesome.min.css" rel="stylesheet" type="text/css">

	
	<link rel="shortcut icon" type="image/x-icon" href="https://cwcyau.github.io/favicon.ico">
	<link rel="icon" type="image/x-icon" href="https://cwcyau.github.io/favicon.ico">
	

	<!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
	<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
	<!--[if lt IE 9]>
	<script src="js/ie/html5shiv.js"></script>
	<script src="js/ie/html5shiv.jsrespond.min.js"></script>
	<![endif]-->
</head>
    <body>

    <!-- Wrapper -->
        <div id="wrapper">

            <!-- Header -->
    <header id="header" class="alt">
        <a href="https://cwcyau.github.io" class="logo"><strong>Statistical Machine Learning</strong> <span>Health &amp; Wellbeing</span></a>
        <nav>
            <a href="#menu">Menu</a>
        </nav>
    </header>

<!-- Menu -->
    <nav id="menu">
        <ul class="links">
            
                <li><a href="https://cwcyau.github.io">Home</a></li>
            
                <li><a href="https://cwcyau.github.io/group.html">About</a></li>
            
                <li><a href="https://cwcyau.github.io/blogs/news">News</a></li>
            
                <li><a href="https://cwcyau.github.io/blogs/research">Research</a></li>
            
                <li><a href="https://cwcyau.github.io/blogs/join">Opportunities</a></li>
            

        </ul>
        <ul class="actions vertical">
            
            
        </ul>
    </nav>

        <!-- Main -->
            <div id="main" class="alt">

                
                    <section id="one">
                        <div class="inner">
                            <header class="major">
                                <h1>Research</h1>
                            </header>
                            
                            

<h2 id="overview">Overview</h2>

<p>Our research is primarily concerned with the development of statistical and machine learning methodologies with a particular focus on applications in the biomedical sciences.</p>

<p>A complete list of publications can be found on <a href="https://scholar.google.co.uk/citations?user=5tyrt68AAAAJ&amp;hl=en">Google Scholar</a> and our research software is available on <a href="https://github.com/cwcyau">Github</a></p>

<h2 id="cancer">Cancer</h2>

<p>Tumour heterogeneity describes the genetic diversity both within (intra-tumour heterogeniety) and between tumours (inter-tumour heterogeneity). Genetic differences within and between tumours give rise to different disease outcomes and patient responses to therapies. Understanding and characterising tumour heterogeneity is therefore important in developing clinical approaches that are tailored to a patient (individualised medicine).</p>

<p>Our group is interested in developing statistical methods to analyse genome sequencing data that comes from heterogeneous tumour samples using advanced machine learning.</p>

<p>Read more about our cancer research and media coverage of our work:</p>

<ul>
<li><a href="http://www.ebiomedicine.com/article/S2352-3964(16)30305-X/abstract">Premalignant SOX2 overexpression in the fallopian tubes of ovarian cancer patients: Discovery and validation studies</a> - <strong>EBioMedicine</strong></li>
<li><a href="http://www.ox.ac.uk/news/2016-07-29-two-research-discoveries-offer-hope-managing-ovarian-cancer">Two research discoveries offer hope for managing ovarian cancer</a> - <strong>University of Oxford</strong></li>
<li><a href="http://www.telegraph.co.uk/science/2016/07/28/ovarian-cancer-test-on-horizon-as-scientists-find-earliest-signs/">Ovarian cancer test on horizon as scientists find earliest signs of disease</a> - <strong>The Telegraph</strong></li>
<li><a href="http://www.dailymail.co.uk/health/article-3712949/Ovarian-cancer-detected-deadly-Scientists-identify-key-enzyme-makes-disease-spread.html">Ovarian cancer can be detected BEFORE it becomes deadly: Scientists identify key enzyme which makes the disease spread</a> - <strong>The Daily Mail</strong></li>
<li><a href="https://ovarian.org.uk/news-and-blog/news/researchers-identify-how-ovarian-cancer-hijacks-natural-cell-process-to-survive/">Researchers identify how ovarian cancer hijacks natural cell process to survive</a> - <strong>Ovarian Cancer Action</strong></li>
<li><a href="https://www.birmingham.ac.uk/news/latest/2017/06/cancer-hijacks-natural-cell-process.aspx">Research finds cancer hijacks natural cell process to survive</a> - <strong>University of Birmingham</strong></li>
</ul>

<h2 id="single-cell-sequencing">Single Cell Sequencing</h2>

<p>Advances in single cell technology now allow large-scale experimentation on single cells in a high-throughput fashion providing new insight into cellular function. Heterogeneity, both biological and technical, confound the simple interpretation of single cell data and sophisticated statistical methods are required to handle different sources of noise and signal.</p>

<p>Our group is working on approaches using Bayesian hierarchical modelling to integrate information from multiple sources across different spatiotemporal scales in order to better understand cellular function and dynamics.</p>

<p>Recent publications:</p>

<ul>
<li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26527291">ZIFA: Dimensionality reduction for zero-inflated single-cell gene expression analysis</a> - <strong>Genome Biology</strong></li>
<li><a href="http://journals.plos.org/plosone/doi?id=10.1371/journal.pcbi.1005212">Order Under Uncertainty: Robust Differential Expression Analysis Using Probabilistic Models for Pseudotime Inference</a> - <strong>PLOS Computational Biology</strong></li>
<li><a href="https://www.nature.com/articles/s41467-018-04696-6">Uncovering pseudotemporal trajectories with covariates from single cell and bulk expression data</a> - <strong>Nature Communications</strong></li>
</ul>

<h2 id="bayesian-statistical-machine-learning">Bayesian Statistical Machine Learning</h2>

<p>Bayesian methods are now ubiquitous in statistical modeling applications across a wide range of disciplines. A major challenge for Bayesian approaches is the significant computation required for exact inference in large models applied to big datasets (terabyte or more scale). For this type of data, Markov Chain Monte Carlo (MCMC) simulation approaches are not feasible despite recent advances in massively parallel computational hardware (e.g. graphics programming units or GPUs). Instead, it is necessary to develop approximate methods that are able to give &ldquo;good&rdquo; answers that, whilst not guaranteed to be exact or optimal, are sufficient for downstream decision processes and further scientific inquiry.</p>

<p>Our group is developing approximate inference methods to fit complex statistical models to large datasets that respect the practical computational and time limitations that govern real-life scientific studies. We are also interested in using decision-theoretic ideas to produce meaningful results for scientists via loss functions that are tailored to the task at hand.</p>

<p>Our recent work in this area has featured in the leading statistical and machine learning journals and conferences:</p>

<ul>
<li><a href="http://proceedings.mlr.press/v80/rukat18a.html">Probabilistic Boolean Tensor Decomposition</a> - <strong>International Conference on Machine Learning</strong></li>
<li><a href="http://papers.nips.cc/paper/6733-testing-and-learning-on-distributions-with-symmetric-noise-invariance">Testing and learning on distributions with symmetric noise invariance</a> - <strong>Advances in Neural Information Processing Systems</strong></li>
<li><a href="http://amstat.tandfonline.com/doi/abs/10.1080/01621459.2016.1222288">The Hamming Ball Sampler</a> - <strong>The Journal of the American Statistical Association</strong></li>
<li><a href="https://papers.nips.cc/paper/5456-hamming-ball-auxiliary-sampling-for-factorial-hidden-markov-models.pdf">Hamming Ball Auxiliary Sampling for Factorial Hidden Markov Models</a> - <strong>Advances in Neural Information Processing Systems (NIPS)</strong></li>
<li><a href="http://www.tandfonline.com/doi/abs/10.1080/01621459.2014.998762">Statistical Inference in Hidden Markov Models Using k-Segment Constraints</a> - <strong>The Journal of the American Statistical Association</strong></li>
</ul>

                        </div>
                    </section>
            <!-- Disqus Inject -->
                
            </div>
            
        <!-- Footer -->
            
                <!-- Footer -->
    <footer id="footer">
        <div class="inner">
            <ul class="icons">
                
                    <li><a href="https://twitter.com/cwcyau" class="icon alt fa-twitter" target="_blank"><span class="label">Twitter</span></a></li>
                
                    <li><a href="https://github.com/cwcyau" class="icon alt fa-github" target="_blank"><span class="label">GitHub</span></a></li>
                
                    <li><a href="https://www.linkedin.com/in/christopher-yau-56227b34/" class="icon alt fa-linkedin" target="_blank"><span class="label">LinkedIn</span></a></li>
                
            </ul>
            <ul class="copyright">
                <li>&copy; Christopher Yau</li>
                
                <li>Design:  <a href="https://www.html5up.net">HTML5 UP</a></li>
                
            </ul>
        </div>
    </footer>

            
        </div>

    <!-- Scripts -->
        <!-- Scripts -->
    <!-- jQuery -->
    <script src="https://cwcyau.github.io/js/jquery.min.js"></script>
    <script src="https://cwcyau.github.io/js/jquery.scrolly.min.js"></script>
    <script src="https://cwcyau.github.io/js/jquery.scrollex.min.js"></script>
    <script src="https://cwcyau.github.io/js/skel.min.js"></script>
    <script src="https://cwcyau.github.io/js/util.js"></script>

    

    <!-- Main JS -->
    <script src="https://cwcyau.github.io/js/main.js"></script>

    

    

    </body>
</html>
