<!DOCTYPE html>
<!-- This site was created with Hugo Blox. https://hugoblox.com -->
<!-- Last Published: December 3, 2025 --><html lang="en-us" >


<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  
  
  
    <meta name="generator" content="Hugo Blox Builder 5.9.7" />
  

  
  












  
  










  







  
  

  
  
  

  
  

  
  

  

  <link rel="stylesheet" href="/css/vendor-bundle.min.26c458e6907dc03073573976b7f4044e.css" media="print" onload="this.media='all'">

  
  
  
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1.9.4/css/academicons.min.css" integrity="sha512-IW0nhlW5MgNydsXJO40En2EoCkTTjZhI3yuODrZIc8cQ4h1XcF53PsqDHa09NqnkXuIe0Oiyyj171BqZFwISBw==" crossorigin="anonymous" media="print" onload="this.media='all'">
    

    
    
    
    
      
      
    
    
    

    
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.min.css" integrity="" crossorigin="anonymous" media="print" onload="this.media='all'">
    

    

    
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      
        
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
  

  
  
  
  
  
  
  <link rel="stylesheet" href="/css/wowchemy.9c85b7451138c421e4165efcc1eee9b5.css" />

  
  
  

  
  
  
  
  
  
  
    
    
    <link rel="stylesheet" href="/css/libs/chroma/github-light.min.css" title="hl-light" media="print" onload="this.media='all'" >
    <link rel="stylesheet" href="/css/libs/chroma/dracula.min.css" title="hl-dark" media="print" onload="this.media='all'" disabled>
  

  
  



























  
  
  






  <meta name="author" content="Christopher Yau" />





  

<meta name="description" content="A highly-customizable Hugo research group theme powered by Wowchemy website builder." />



<link rel="alternate" hreflang="en-us" href="https://cwcyau.github.io/category/cancer/" />
<link rel="canonical" href="https://cwcyau.github.io/category/cancer/" />



  <link rel="manifest" href="/manifest.webmanifest" />



<link rel="icon" type="image/png" href="/media/icon_hu_2311fe1a408f47c1.png" />
<link rel="apple-touch-icon" type="image/png" href="/media/icon_hu_4eb7679362359773.png" />

<meta name="theme-color" content="#1565c0" />










  
  






<meta property="twitter:card" content="summary" />

  <meta property="twitter:site" content="@GetResearchDev" />
  <meta property="twitter:creator" content="@GetResearchDev" />
<meta property="twitter:image" content="https://cwcyau.github.io/media/icon_hu_7613a4a452ac7087.png" />



  

<meta property="og:type" content="website" />
<meta property="og:site_name" content="Yau Research Group" />
<meta property="og:url" content="https://cwcyau.github.io/category/cancer/" />
<meta property="og:title" content="Cancer | Yau Research Group" />
<meta property="og:description" content="A highly-customizable Hugo research group theme powered by Wowchemy website builder." /><meta property="og:image" content="https://cwcyau.github.io/media/icon_hu_7613a4a452ac7087.png" /><meta property="og:locale" content="en-us" />

  
    <meta property="og:updated_time" content="2025-07-11T00:00:00&#43;00:00" />
  










  
  
  

  
  
    <link rel="alternate" href="/category/cancer/index.xml" type="application/rss+xml" title="Yau Research Group" />
  

  


  
  <title>Cancer | Yau Research Group</title>

  
  
  
  











</head>


<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" class="page-wrapper   "  >

  
  
  
  
  
  
  
  
  
  <script src="/js/wowchemy-init.min.4fef3e534144e9903491f0cc6527eccd.js"></script>

  


<aside class="search-modal" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#" aria-label="Close"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search" class="form-control"
        aria-label="Search...">
        
      </div>

      
      

      

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>



  <div class="page-header header--fixed">
  
  
  
  
  












<header>
  <nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
    <div class="container-xl">

      
      <div class="d-none d-lg-inline-flex">
        <a class="navbar-brand" href="/">Yau Research Group</a>
      </div>
      

      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar-content" aria-controls="navbar-content" aria-expanded="false" aria-label="Toggle navigation">
      <span><i class="fas fa-bars"></i></span>
      </button>
      

      
      <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
        <a class="navbar-brand" href="/">Yau Research Group</a>
      </div>
      

      
      
      <div class="navbar-collapse main-menu-item collapse justify-content-end" id="navbar-content">

        
        <ul class="navbar-nav d-md-inline-flex">
          

          

          
          
          

          

          
          
          
          

          
            
              
              
            
            
          

          <li class="nav-item">
            <a class="nav-link " href="/tour"><span>About</span></a>
          </li>

          
          

          

          
          
          

          

          
          
          
          

          
            
              
              
            
            
          

          <li class="nav-item">
            <a class="nav-link " href="/post"><span>News</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
          

          <li class="nav-item">
            <a class="nav-link " href="/people"><span>People</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
          

          <li class="nav-item">
            <a class="nav-link " href="/event"><span>Projects</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
          

          <li class="nav-item">
            <a class="nav-link " href="/publication"><span>Publications</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
          

          <li class="nav-item">
            <a class="nav-link " href="/contact"><span>Contact</span></a>
          </li>

          
          

        

          
        </ul>
      </div>

      <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">

        
        
          
        

        
        
        
        <li class="nav-item">
          <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        
        
        

        
        

      </ul>

    </div>
  </nav>
</header>


  </div>

  <div class="page-body">
    
    
    

    













  

  
  
  
    
  
<div class="universal-wrapper pt-3">
  <h1>Cancer</h1>

  

  
</div>



<div class="universal-wrapper">
  


  

  
  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/2025-07-01_gperturb/" >GPerturb in Nature Communications</a>
    </div>

    
    <a href="/post/2025-07-01_gperturb/"  class="summary-link">
      <div class="article-style">
        <p>Congratulations to <a href="../../authors/xing">Hanwen Xing</a> on having his paper <a href="https://www.nature.com/articles/s41467-025-61165-7" target="_blank" rel="noopener">&ldquo;GPerturb: Gaussian process modelling of single-cell perturbation data&rdquo;</a> accepted in Nature Communications:</p>
<p>&ldquo;Single-cell RNA sequencing and CRISPR screening enable high-throughput analysis of genetic perturbations at single-cell resolution. Understanding combinatorial perturbation effects is essential but challenging due to data sparsity and complex biological mechanisms. We present GPerturb, a Gaussian process-based sparse perturbation regression model designed to estimate gene-level perturbation effects. GPerturb employs an additive structure to separate signal from noise and captures sparse, interpretable effects from both discrete and continuous responses. It also provides uncertainty estimates for the presence and strength of perturbation effects on individual genes. We demonstrate the use GPerturb on both simulated and real-world datasets, characterising its competitive performance with current state-of-the-art methods. Furthermore, the model reveals meaningful gene-perturbation interactions and identifies effects consistent with known biology. GPerturb offers a novel approach for uncovering complex dependencies between gene expression and perturbations and advancing our understanding of gene regulation at the single-cell level.&rdquo;</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Jul 11, 2025
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    1 min read
  </span>
  

  
  
  
  

  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fas fa-folder mr-1"></i><a href="/category/machine-learning/">Machine Learning</a>, <a href="/category/cancer/">Cancer</a></span>
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/2024-08-16_mlcb_2024/" >MLCB 2024</a>
    </div>

    
    <a href="/post/2024-08-16_mlcb_2024/"  class="summary-link">
      <div class="article-style">
        <p>Congratulations to <a href="../../authors/gadd">Charles Gadd</a> on having his paper accepted at <a href="https://sites.google.com/cs.washington.edu/mlcb2024/home?authuser=0" target="_blank" rel="noopener">MLCB 2024</a>.</p>
<p>Changes in the number of copies of certain parts of the genome, known as copy number alterations (CNAs), due to somatic mutation processes are a hallmark of many cancers. This genomic complexity is known to be associated with poorer outcomes for patients but describing its contribution in detail has been difficult. Copy number alterations can affect large regions spanning whole chromosomes or the entire genome itself but can also be localised to only small segments of the genome and no methods exist that allow this multi-scale nature to be quantified. In this paper, we address this using Wave-LSTM, a signal decomposition approach designed to capture the multi-scale structure of complex whole genome copy number profiles. Using wavelet-based source separation in combination with deep learning-based attention mechanisms. We show that Wave-LSTM can be used to derive multi-scale representations from copy number profiles which can be used to decipher sub-clonal structures from single-cell copy number data and to improve survival prediction performance from patient tumour profiles.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Aug 16, 2024
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    1 min read
  </span>
  

  
  
  
  

  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fas fa-folder mr-1"></i><a href="/category/machine-learning/">Machine Learning</a>, <a href="/category/cancer/">Cancer</a></span>
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/2023-12-13_rarity_2023/" >New paper in Bioinformatics</a>
    </div>

    
    <a href="/post/2023-12-13_rarity_2023/"  class="summary-link">
      <div class="article-style">
        <p>Congratulations to <a href="../../authors/martens">Kaspar Martens</a> on having his paper accepted in <a href="https://academic.oup.com/bioinformatics/article/39/12/btad750/7471872?login=false" target="_blank" rel="noopener">Bioinformatics</a>.</p>
<p>Cell type identification plays an important role in the analysis and interpretation of single-cell data and can be carried out via supervised or unsupervised clustering approaches. Supervised methods are best suited where we can list all cell types and their respective marker genes a priori, while unsupervised clustering algorithms look for groups of cells with similar expression properties. This property permits the identification of both known and unknown cell populations, making unsupervised methods suitable for discovery. Success is dependent on the relative strength of the expression signature of each group as well as the number of cells. Rare cell types therefore present a particular challenge that is magnified when they are defined by differentially expressing a small number of genes.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Dec 13, 2023
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    2 min read
  </span>
  

  
  
  
  

  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fas fa-folder mr-1"></i><a href="/category/machine-learning/">Machine Learning</a>, <a href="/category/cancer/">Cancer</a></span>
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/2023-11-30_picocnv_2023/" >New publication in BMC Bioinformatics</a>
    </div>

    
    <a href="/post/2023-11-30_picocnv_2023/"  class="summary-link">
      <div class="article-style">
        <p>Congratulations to <a href="../../authors/nulsen">Joel Nulsen</a> on having his paper accepted in <a href="https://link.springer.com/article/10.1186/s12859-023-05576-7" target="_blank" rel="noopener">BMC Bioinformatics</a>.</p>
<p>Genomic insights in settings where tumour sample sizes are limited to just hundreds or even tens of cells hold great clinical potential, but also present significant technical challenges. We previously developed the DigiPico sequencing platform to accurately identify somatic mutations from such samples.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Nov 30, 2023
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    1 min read
  </span>
  

  
  
  
  

  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fas fa-folder mr-1"></i><a href="/category/machine-learning/">Machine Learning</a>, <a href="/category/cancer/">Cancer</a></span>
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/2023-08-16_mlcb_2023/" >MLCB 2023</a>
    </div>

    
    <a href="/post/2023-08-16_mlcb_2023/"  class="summary-link">
      <div class="article-style">
        <p>Congratulations to <a href="../../authors/martens">Kaspar Martens</a> on having his paper accepted at <a href="https://sites.google.com/cs.washington.edu/mlcb2023/" target="_blank" rel="noopener">MLCB 2023</a>.</p>
<p>Generative models for multimodal data permit the identification of latent factors that may be associated with important determinants of observed data heterogeneity. Common or shared factors could be important for explaining variation across modalities whereas other factors may be private and important only for the explanation of a single modality. Multimodal Variational Autoencoders, such as MVAE and MMVAE, are a natural choice for inferring those underlying latent factors and separating shared variation from private. In this work, we investigate their capability to reliably perform this disentanglement. In particular, wehighlight a challenging problem setting where modality-specific variation dominates the shared signal. Taking a cross-modal prediction perspective, we demonstrate limitations of existing models, and propose a modification how to make them more robust to modalityspecific variation. Our findings are supported by experiments on synthetic as well as various real-world multi-omics data sets. The paper is available on <a href="https://proceedings.mlr.press/v240/martens24a.html" target="_blank" rel="noopener">PMLR</a>.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Aug 16, 2023
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    1 min read
  </span>
  

  
  
  
  

  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fas fa-folder mr-1"></i><a href="/category/machine-learning/">Machine Learning</a>, <a href="/category/cancer/">Cancer</a></span>
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/mycancerai/" >New report: My Cancer &amp; AI</a>
    </div>

    
    <a href="/post/mycancerai/"  class="summary-link">
      <div class="article-style">
        <p>A report on a patient engagement workshop series on what cancer patients think about artificial intelligence is now available.</p>
<p>The workshop series was developed in partnership with <a href="https://ovarian.org.uk/" target="_blank" rel="noopener">Ovarian Cancer Action</a> (OCA) as part of my Turing AI Fellowship.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Jun 22, 2022
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    1 min read
  </span>
  

  
  
  
  

  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fas fa-folder mr-1"></i><a href="/category/machine-learning/">Machine Learning</a>, <a href="/category/cancer/">Cancer</a></span>
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
      
      
      <a href="/post/mycancerai/" >
        <img src="/post/mycancerai/featured_hu_3a78caa3d5cf137a.webp" height="35" width="150"
            alt="New report: My Cancer &amp; AI" loading="lazy">
      </a>
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/2021-12-20_gbio/" >CIDER: an interpretable meta-clustering framework for single-cell RNA-seq data integration and evaluation</a>
    </div>

    
    <a href="/post/2021-12-20_gbio/"  class="summary-link">
      <div class="article-style">
        <p>Our latest paper, based on the thesis work of former DPhil student Zhiyuan, has been published in Genome Biology. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02561-2" target="_blank" rel="noopener">CIDER: an interpretable meta-clustering framework for single-cell RNA-seq data integration and evaluation</a> describes a meta-clustering workflow based on inter-group similarity measures. We demonstrate that CIDER outperforms other scRNA-Seq clustering methods and integration approaches in both simulated and real datasets. Moreover, we show that CIDER can be used to assess the biological correctness of integration in real datasets, while it does not require the existence of prior cellular annotations.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Dec 20, 2021
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    1 min read
  </span>
  

  
  
  
  

  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fas fa-folder mr-1"></i><a href="/category/cancer/">Cancer</a>, <a href="/category/omics/">Omics</a></span>
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/2021-12-14_rarity/" >Rarity: Discovering rare cell populations from single-cell imaging data</a>
    </div>

    
    <a href="/post/2021-12-14_rarity/"  class="summary-link">
      <div class="article-style">
        <p>Kaspar Martens will present his latest work at the NeurIPS Workshop &ldquo;Learning Meaningful Representations of Life&rdquo;
<a href="https://drive.google.com/file/d/1r9TINxfXgB_cs4bOcT7ZYZjF5IMcb4X6/view" target="_blank" rel="noopener">Rarity: Discovering rare cell populations from single-cell imaging data</a>. The work arises from his Turing-Crick Biomedical Award which supports a collaboration between the Alan Turing Institute and the Ciccarelli Group at Kings College London and the Francis Crick Institute.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Dec 14, 2021
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    1 min read
  </span>
  

  
  
  
  

  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fas fa-folder mr-1"></i><a href="/category/cancer/">Cancer</a>, <a href="/category/omics/">Omics</a></span>
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/2021-11-20_chfwrf/" >Yau receives award for ovarian cancer research</a>
    </div>

    
    <a href="/post/2021-11-20_chfwrf/"  class="summary-link">
      <div class="article-style">
        <p>The <a href="https://www.chfwrf.org/info/" target="_blank" rel="noopener">Columbia Hospital For Women Research Foundation</a> have awarded <a href="../../authors/admin">Christopher Yau</a> the prize for most impactful paper in 2020 in the field of obstetrical and gynecologic and breast disease. The prize consisted of a $5,000 to a charity of Chris and Ahmed&rsquo;s choosing and they selected <a href="https://ovarian.org.uk/about-us/about-oca/" target="_blank" rel="noopener">Ovarian Cancer Action</a> who co-funded the original work.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Nov 20, 2021
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    1 min read
  </span>
  

  
  
  
  

  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fas fa-folder mr-1"></i><a href="/category/cancer/">Cancer</a></span>
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/2021-11-10_oca_video/" >Artificial Intelligence Webinar - Ovarian Cancer Action UK</a>
    </div>

    
    <a href="/post/2021-11-10_oca_video/"  class="summary-link">
      <div class="article-style">
        <p>Recently <a href="../../authors/admin">Christopher Yau</a> worked with <a href="https://ovarian.org.uk/" target="_blank" rel="noopener">Ovarian Cancer Action UK</a> to put together a webinar on his research for patients and the public. You can find the video on Youtube: <a href="https://www.youtube.com/watch?v=kQ8m4L15GYM" target="_blank" rel="noopener">&ldquo;What is artificial intelligence and what does it mean for cancer research?&rdquo;</a>.</p>
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Nov 10, 2021
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    1 min read
  </span>
  

  
  
  
  

  
  
  <span class="middot-divider"></span>
  <span class="article-categories">
    <i class="fas fa-folder mr-1"></i><a href="/category/machine-learning/">Machine Learning</a>, <a href="/category/cancer/">Cancer</a></span>
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  

  

  
<nav class="mt-1">
  <ul class="pagination justify-content-center">
    
    
    <li class="page-item"><a class="page-link" href="/category/cancer/page/2/">&raquo;</a></li>
    
  </ul>
</nav>


</div>
  </div>

  <div class="page-footer">
    
    
    <div class="container">
      <footer class="site-footer">

  












  
  
  
  
  













  
  
  

  
  
    
  
  
    
  

  

  
  <p class="powered-by copyright-license-text">
    © 2025 Me. This work is licensed under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0" rel="noopener noreferrer" target="_blank">CC BY NC ND 4.0</a>
  </p>
  

  <p class="powered-by footer-license-icons">
    <a href="https://creativecommons.org/licenses/by-nc-nd/4.0" rel="noopener noreferrer" target="_blank" aria-label="Creative Commons">
      <i class="fab fa-creative-commons fa-2x" aria-hidden="true"></i>
      <i class="fab fa-creative-commons-by fa-2x" aria-hidden="true"></i>
      
        <i class="fab fa-creative-commons-nc fa-2x" aria-hidden="true"></i>
      
      
        <i class="fab fa-creative-commons-nd fa-2x" aria-hidden="true"></i>
      
    </a>
  </p>





  <p class="powered-by">
    
    
    
      
      
      
      
      
      
      Published with <a href="https://hugoblox.com/?utm_campaign=poweredby" target="_blank" rel="noopener">Hugo Blox Builder</a> — the free, <a href="https://github.com/HugoBlox/hugo-blox-builder" target="_blank" rel="noopener">open source</a> website builder that empowers creators.
    
  </p>
</footer>

    </div>
    
  </div>

  


<script src="/js/vendor-bundle.min.0dfcdd67ca42b8dbc94d6bf1aa446ffb.js"></script>




  

  
  

  






  <script src="https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.min.js" integrity="" crossorigin="anonymous"></script>








  
  <script id="search-hit-fuse-template" type="text/x-template">
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script>
  
    <script src="https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js" integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js" integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin="anonymous"></script>
  












  
  
  
  
  
  
  

















<script id="page-data" type="application/json">{"use_headroom":true}</script>


  <script src="/js/wowchemy-headroom.db4755770454eb63685f8de785c0a172.js" type="module"></script>









  
  


<script src="/en/js/wowchemy.min.69fc52a6dc16aaf1a452cfbb36466dac.js"></script>



  <script src="/js/wowchemy-map.a26e9d2f7238ba5b868384f1c5bc6477.js" type="module"></script>




  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        
        <pre><code></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>


  <script src="/js/wowchemy-publication.9c0e895144aef5a693008b5c5d450147.js" type="module"></script>


















</body>
</html>
